docs/docs/middlewares/path.md

```diff
@@ -0,0 +1,41 @@
+# Path
+This function returns the URL for the given route. 
+
+## Parameters
+```path(page, queries, locale)```
+
+parameter        | type           | required | description
+-----------------|----------------|----------|------------
+page             | string         | ✓        | The name of page (e.g. www.mysite.com).
+queries          | array          | ❌        | Array of url parameters (e.g. [gallery: nature])
+locale           | string         | ❌        | Shortcut for language.
```

Comment:
It’s not quite a shortcut, it’s rather language code, eg. `cs_CZ` or `en_US`. There is a standard that defines it but I don’t remember its name.

Reply:
I suppose you meant [locale](!https://www.ibm.com/support/knowledgecenter/en/SSLTBW_2.3.0/com.ibm.zos.v2r3.cbcpx01/locnamc.htm). If I understand correctly, 'cz_CZ' is 'Language-Territory'. I'm not sure if 'locale' or 'language code' will be correctly understood. I suggest let the parameter name 'locale' but mention it in the description that it means Language-Territory code (eg. cz_CZ).

Reply:
Locale format using underscore character is known as "Base locale ID" in common. But in this case and for better usability I would suggest: "Language localization ID".